scripts/live-connector-e2e/run-windows.ps1

```
@@ -0,0 +1,344 @@
+    if (Test-Path -LiteralPath $block) { throw 'blocked action executed and created its sentinel' }
+    if (-not (Test-ConnectorEvent $script:GatewayJsonl $Connector $before)) { throw 'blocked tool hook did not reach the gateway' }
+    if (-not (Test-BlockVerdict $script:GatewayJsonl $before)) { throw 'blocked action has no block verdict' }
+    Write-Result tool-block:enforced pass 'sentinel absent and block verdict present'
+    if (-not (Test-OtlpEvent $script:GatewayJsonl $Connector $start)) { throw 'no connector-tagged OTLP telemetry reached the gateway' }
+    Write-Result otlp pass
+    Assert-Evidence $start
+    Invoke-Teardown
+    Write-Result teardown pass
+}
+
+function Stop-IsolatedProcesses {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Stop-IsolatedProcesses' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)